INDEX.HTML

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<title>Hello Microverse</title>
+        <link rel="stylesheet" href="style.css">
+        <title>hello hello</title>
```

Comment:
- [ ] You have the title tag twice which might be causing the lighthouse error please remove one of them.
- [ ] Please fix the lighthouse linter error.

![image](https://user-images.githubusercontent.com/59416136/154290654-d4332805-0a36-4f9e-aadf-460d4b661691.png)

Reply:
this has been a problem I've been  dealing with for sometime

Reply:
Its done but the error persist